tests/phpunit/api/v3/ProfileTest.php

```diff
@@ -99,28 +93,26 @@ public function testProfileGet(): void {
    *
    * We are checking that there is no error.
    *
-   * @throws \CRM_Core_Exception
    */
   public function testProfileGetWithAddressCustomData(): void {
-    $this->_createIndividualContact();
-    $this->entity = 'Address';
```

Comment:
@braders - jenkins accepted it :-) The value passed in for `extends` overwrites the other - it might still cause php 8.2 test fails but they can be fixed at the trait level now

Reply:
Aah, I see now. Passing in `['extends' => 'Address']` is much nicer, but confusing having two ways to achieve the same thing. I can see how removing all the examples of `$this->entity` will make things much clearer.

Reply:
I think the `$this->entity` approach dates back to some ideas we had a long time ago & the `SyntaxConformanceTest` kinda took over from where we were going with that so I think it could be used a lot less. (I think there are some valid uses still)